scene/resources/curve.h

```diff
@@ -280,6 +284,7 @@ class Curve3D : public Resource {
 	Vector3 _sample_baked(Interval p_interval, bool p_cubic) const;
 	real_t _sample_baked_tilt(Interval p_interval) const;
 	Basis _sample_posture(Interval p_interval, bool p_apply_tilt = false) const;
+	Basis _compose_posture(int p_baked_idx) const;
```

Comment:
```suggestion
	Basis _compose_posture(int p_index) const;
```

Reply:
done

Reply:
This method is hidden in gdscript and only the core developer cares about it, it doesn't need to be very explicit, and I think it's more important for it to look consistent. If you are concerned, you can annotate it as a comment.

Reply:
Also for `Basis Curve3D::get_point_baked_posture(int p_idx, bool p_apply_tilt)`

Reply:
Fair enough, done